plugins/src/webrtcsink/imp.rs

```diff
@@ -330,6 +330,10 @@ fn configure_encoder(enc: &gst::Element) {
                 enc.set_property_from_str("rc-mode", "cbr-ld-hq");
                 enc.set_property("zerolatency", true);
             }
+            "vaapih264enc" | "vaapivp8enc" => {
+                enc.set_property("bitrate", 2048u32);
+                enc.set_property("keyframe-period", 2560u32);
+            }
```

Comment:
I noticed that the default values for bitrate and keyframe distance differ between h264 and VP8 (taking the different unit  (Bit/s, KBit/s into account) even though bitrate for example is set for both encodings the same way in latter functions. However as both encoders have the same properties I grouped them here.

The latency seemed rather low though I only checked the delay between my PC clock and `clockoverlay` as I did not have any better tooling for latency evaluation.
I am not sure if setting constant bitrate (`cbr`) mode yields any benefits as this denies the codec the possibility to use a lower bitrate if feasible (at least that is my understanding—again I do not know too much about gstreamer or encoders ein general)

Reply:
So, the idea with the implementation of congestion control is that the algorithm is constantly trying to probe the available bandwidth by raising the bitrate (slowly when it considers that it is within a window where congestion was previously observed, then more rapidly once out of that window). This doesn't work well if for some reason (eg static scene or what have you) the encoder decides it doesn't actually need to use as much bitrate as requested, and that's why using CBR can help by making things more predictable :)

Reply:
Ah that makes sense. I added that setting now.